test/sandbox/contract/printEncodedFunctions.ts

```diff
@@ -10,8 +10,8 @@ require('dotenv').config()
  */
 const log = new Logger('sandbox:printEncodedFunctions')
 
-async function exec (): Promise<void> {
-  const abi = require('../../../src/contracts/StablecoinConverter.json')
+async function exec(): Promise<void> {
+  const abi = require('contracts/StablecoinConverter.json')
```

Comment:
Can you `import` the json instead of `require` ?
I remember es-lint having a warning about that.

Reply:
can't dynamically import like that